hscontrol/db/preauth_keys.go

```diff
@@ -149,14 +172,19 @@ func ListPreAuthKeysByUser(tx *gorm.DB, uid types.UserID) ([]types.PreAuthKey, e
 	}
 
 	keys := []types.PreAuthKey{}
-	if err := tx.Preload("User").Where(&types.PreAuthKey{UserID: user.ID}).Find(&keys).Error; err != nil {
+
+	err = tx.Preload("User").Where(&types.PreAuthKey{UserID: &user.ID}).Find(&keys).Error
+	if err != nil {
 		return nil, err
 	}
 
 	return keys, nil
 }
 
-var ErrPreAuthKeyFailedToParse = errors.New("failed to parse AuthKey")
+var (
+	ErrPreAuthKeyFailedToParse    = errors.New("failed to parse AuthKey")
+	ErrPreAuthKeyNotTaggedOrOwned = errors.New("preauthkey must be either tagged or owned by user")
```

Comment:
I think we should style it `auth-key` anywhere user facing, so keep it the same in the code, variable, types and such, but any user facing messaging and docs should be `auth-key`, not:
- AuthKey
- PreAuthKey
- pre-auth-key
- preauthkey
- authkey
- preauth-key
- pre-authkey

and so on.